repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -747,9 +749,11 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
 
 object KeystoneV2Filter {
 
-  private final val SYSTEM_MODEL_CONFIG = "system-model.cfg.xml"
-  private final val DEFAULT_CONFIG = "keystone-v2.cfg.xml"
-  private final val X_AUTH_PROXY = "Proxy"
+  private final val SystemModelConfig = "system-model.cfg.xml"
+  private final val DefaultConfig = "keystone-v2.cfg.xml"
+  private final val XAuthProxy = "Proxy"
```

Comment:
It's particularly annoying when you are trying to use Java and Scala libraries together. `ThatConstant` and `THAT_OTHER_CONSTANT`. Yay!